### PR TITLE
Import automation: limiting and sorting system runs returned

### PR DESCRIPTION
**Pull request by intrepiditee**
This PR adds the capability to limit the number of system runs returned and sort them by some fields.

Thanks for reviewing.

### CODE REVIEW

File: import-automation/progress-dashboard-rest/app/resource/system_run_list.py [hunk 1]
```diff
@@ -28,24 +32,40 @@ class SystemRunList(system_run.SystemRun):
...
     def get(self):
         """Retrieves a list of system runs that pass the filter defined by
         the key-value mappings in the request body.
 
         The filter can only contain fields defined by SystemRun.
 
+        This endpoint accepts two url arguments: limit and order.
+        limit is an integer that specifies the maximum number of system runs
+        returned and order is a list of field names to order the returned
+        system runs by. Prepend - to a field name to sort it in
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app/resource/system_run_list.py:48]:
```suggestion
        system runs by. Prepend "-" to a field name to sort it in
```

File: import-automation/progress-dashboard-rest/app/service/base_database.py [hunk 2]
```diff
@@ -106,14 +106,18 @@ def filter(self, kv_dict):
 
         Args:
             kv_dict: Key-value mappings used for filtering as a dict.
+            order: Sequence of field names each as a string to order
+                the returned entities. Prepend - to a field name to sort it
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app/service/base_database.py:110]: Likewise.

File: import-automation/progress-dashboard-rest/app/service/base_database.py [hunk 3]
```diff
@@ -106,14 +106,18 @@ def filter(self, kv_dict):
 
         Args:
             kv_dict: Key-value mappings used for filtering as a dict.
+            order: Sequence of field names each as a string to order
```
Comment (jeffreyoldham) [inline import-automation/progress-dashboard-rest/app/service/base_database.py:109]: I think you are specifying a tuple, not a sequence.

Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/service/base_database.py:109]: fixed

File: import-automation/progress-dashboard-rest/app/service/base_database.py (hunk 2 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/service/base_database.py:110]: fixed

File: import-automation/progress-dashboard-rest/app/resource/system_run_list.py (hunk 1 above)
Comment (intrepiditee) [reply import-automation/progress-dashboard-rest/app/resource/system_run_list.py:48]: fixed

**Pull request merged**